Xamarin.Forms.Platform.Android/FastRenderers/AutomationPropertiesProvider.cs

```diff
@@ -30,52 +31,96 @@ internal static void GetDrawerAccessibilityResources(global::Android.Content.Con
 			resourceIdClose = context.Resources.GetIdentifier($"{automationIdParent}{s_defaultDrawerIdCloseSuffix}", "string", context.ApplicationInfo.PackageName);
 		}
 
-
-		internal static void SetAutomationId(AView control, Element element, string value = null)
+		static bool ShoudISetImportantForAccessibilityToNoIfAutomationIdIsSet(AView control, Element element)
 		{
-			if (!control.IsAlive() || element == null)
+			if (!Flags.IsAccessibilityExperimentalSet())
+				return false;
+
+			// User has specifically said what they want
+			if (AutomationProperties.GetIsInAccessibleTree(element) == true)
+				return false;
+
+			// User has explicitly set name and help text so we honor that
+			if (!String.IsNullOrWhiteSpace(ConcatenateNameAndHelpText(element)))
+				return false;
+
+			if (element is Layout ||
```

Comment:
This set of controls isn't going to change

This is a bug that's very specific to XF 5 and won't surface in MAUI as we won't be using ContentDescription